scripts/ansible/roles/windows/jenkins/tasks/environment-setup.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please create a new role called `openenclave` and have the `environment-setup.yml` there, since this is the actually the env setup for OE, and not Jenkins.